test/khash_test.erl

```diff
@@ -0,0 +1,443 @@
+-module(khash_test).
+
+-compile([export_all]).
+
+-define(NODEBUG, true).
```

Comment:
I can reproduce the issue with 18.0.2. Apparently R18 compiler ignores `export_all` flag, not sure by design or is it a bug.

I'll remove `NODEBUG` just to simplify things. Thank you for checking this for me.